databricks_sql_connector_core/tests/e2e/common/decimal_tests.py

```diff
@@ -1,14 +1,28 @@
 from decimal import Decimal
 
-import pyarrow
+try:
+    import pyarrow
+except ImportError:
+    pyarrow = None
 import pytest
 
 
-class DecimalTestsMixin:
-    decimal_and_expected_results = [
+def decimal_and_expected_results():
+    if pyarrow is None:
+        return []
+
```

Comment:
If the whole test suite is skipped when pyarrow not available - no need to ass additional checks to these functions as they will not be executed anyway (same for other helper functions here)

Reply:
fixed